### PR TITLE
Add idna version info to requests.help

### LINKED ISSUE #4180: support idna.__version__
https://github.com/kjd/idna/issues/49#event-1142557883

### PR DESCRIPTION
**Pull request by alexwlchan**
Resolves #4180.

### CODE REVIEW

**Comment by alexwlchan** [comment]
I’d try to fix the failing tests, but I can’t see the Jenkins instance so I don’t know what’s gone wrong. 😕 

(My two new tests seem like obvious culprits, but they’re passing locally.)

**Comment by nateprewitt** [comment]
Hey @alexwlchan, the two new tests are failing but I don't believe it's related to your patch. Jenkins is complaining on Python 2.6 that `AttributeError: 'module' object has no attribute 'OPENSSL_VERSION_NUMBER'`.

The `info()` function was never tested before, and `OPENSSL_VERSION_NUMBER` doesn't appear to exist on the system [`ssl`](https://github.com/python/cpython/blob/2.6/Lib/ssl.py) module for Python 2.6. We'll need to fix that to get the tests passing. If you feel like throwing together another PR to fix that, it would be greatly appreciated. Otherwise, I'll put something together this evening.

**Comment by alexwlchan** [comment]
> If you feel like throwing together another PR to fix that, it would be greatly appreciated.

It’s five past midnight here, so not right now. Maybe in the morning. 😉

**Comment by nateprewitt** [comment]
No problem at all, @alexwlchan :) I've opened #4183 to address the test issue. Once we get another maintainer to green light that, we should be set here.

**Comment by alexwlchan** [comment]
Okay, rebased against master, so (fingers crossed) this time the tests will pass.

**Comment by Lukasa** [comment]
Well this test output seems extremely problematic. @kennethreitz any reason all the tests are failing to build now?

**Comment by nateprewitt** [comment]
OK, it looks like the builds are working again. @alexwlchan, would you mind rebasing this onto the current master branch?

**Comment by alexwlchan** [comment]
> would you mind rebasing this onto the current master branch?

Done.

File: tests/test_help.py [hunk 1]
```diff
@@ -19,3 +19,22 @@ def test_system_ssl_py26():
...
     assert info()['system_ssl']['version'] != ''
+
+
+class VersionedPackage(object):
+    def __init__(self, version):
+        self.__version__ = version
+
+
+def test_idna_without_version_attribute(mocker):
+    """Older versions of IDNA don't provide a __version_ attribute, verify
```
Comment (nateprewitt) [inline tests/test_help.py]: Sorry, one final note and I think we're set here. Could we get the second underscore for the closing dunder on version (`__version_` -> `__version__`). Once that's done I think I'm happy with this.

Comment (alexwlchan) [reply tests/test_help.py]: Fixed.

**Comment by kennethreitz** [comment]
Sorry for the CI issue, I didn't see this notification.

**Comment by kennethreitz** [comment]
Let me know if it happens again!

**Comment by nateprewitt** [comment]
@kennethreitz, no worries 😄!

**Pull request merged**